app/controllers/invitations_controller.rb

```diff
@@ -0,0 +1,44 @@
+class InvitationsController < ApplicationController
+
+  before_action :require_user, only: [:new, :create]
+
+  def new
+    @invitation = Invitation.new
+    @invitation.user = current_user
+  end
+
+  def create
+    params = invitation_params
```

Comment:
fixed.
The reason for that is that I don't like to go through the invitation_params method over and over. IMHO it's not necessary to check the permitted parameters 4 or 5 times inside one create request.